integration-tests/tests/storage/document-collection.spec.ts

```diff
@@ -0,0 +1,487 @@
+/* eslint-disable no-process-env */
+// eslint-disable-next-line import/no-extraneous-dependencies
+import 'reflect-metadata';
+import { ProjectType } from 'testkit/gql/graphql';
+import { initSeed } from 'testkit/seed';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { createStorage } from '@hive/storage';
+import { crypto } from '@whatwg-node/fetch';
+
+const connectionString = `pg://${process.env.POSTGRES_USER!}:${process.env
+  .POSTGRES_PASSWORD!}@localhost:5432/${process.env.POSTGRES_DB!}`;
+
+const makeStorage = () => {
+  return createStorage(connectionString, 1);
+};
+
+describe('document collection', () => {
```

Comment:
Added these tests to verify that db operations work as expected. Feel free to convert this to GraphQL API tests (similar to https://github.com/kamilkisiela/graphql-hive/blob/03359b8093bd4683740b739ea61cc610ff231997/integration-tests/tests/api/artifacts-cdn.spec.ts#L616)